app/views/user_mailer/welcome_email.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1><%= "Ruby on Rails Training"%></h1>
+    <p>
+    <b><%= t I18n.t('.email.created_at') %> </b> <%= @rent.created_at %><br>
+      <b><%= t I18n.t('.book.details') %> </b>
+      <br>
+          <b><%= t  I18n.t('.book.gender') %> </b><%= @book.gender %><br>
```

Comment:
remember to remove the `I18n.t` and just keep it as t 'key'.

Also, keys should be without an starting dot:

`<%= t I18n.t('.email.created_at') %> ` should be `<%= t 'email.created_at' %> ` instead.

Reply:
done!